### PR TITLE
Read me changes

### PR DESCRIPTION
**Pull request by ComlOnline**
Could someone give this a once over and make sure it all reads correctly, 

Cheers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,19 +8,13 @@ which are not available in the official library.
 
 Note: librespot only works with Spotify Premium
 
-# THIS FORK
-As the origin is no longer maintained I wanted to have a place for a version of librespot with other peoples forks and features merged.
+## This fork
+As the origin by [plietar](https://github.com/plietar/) is no longer actively maintained I wanted to have a place for a version of librespot with other peoples forks and features merged.
 
+# Wiki
+More information can befound in the [wiki](https://github.com/librespot-org/librespot/wiki)
```
Comment (alin23) [inline README.md]: Typo: _befound_

Comment (ComlOnline) [reply README.md]: Thank you! I knew there would be some thing I missed.

**Comment by ComlOnline** [comment]
@alin23 Does it all seem logical? I wanted the ReadMe to be more of a quick overview with the Wiki being the main reference.

**Comment by alin23** [comment]
@ComlOnline Yes, I like the way you did it. This way we can expand the documentation on more complex topics (e.g. cross-compiling) without making the README an unreadable mess.
We should merge it 💯

**Pull request merged**